src/main/java/net/pms/util/FileUtil.java

```diff
@@ -1224,14 +1227,12 @@ public static String[] getFileNameMetadata(String filename, String absolutePath)
 			if (tvSeason.length() > 1 && tvSeason.startsWith("0")) {
 				tvSeason = tvSeason.substring(1);
 			}
-			String showNameIndexMatcher = "(?i) (S\\d{2}E\\d{2}|S\\d{2}|S\\d{2}E\\d{2}-\\d{2}|\\d{4}/\\d{2}/\\d{2})";
-			pattern = Pattern.compile(showNameIndexMatcher + " - (.*)");
-			int showNameIndex = indexOf(pattern, formattedName);
+			int showNameIndex = indexOf(SHOW_NAME_INDEX_PATTERN, formattedName);
```

Comment:
Maybe this line can be moved after the next line `if (isEmpty(movieOrShowName)) {`
as it is used only on this condition (don't process something we don't use).